src/main/java/com/nutomic/syncthingandroid/syncthing/NetworkReceiver.java

```diff
@@ -13,19 +14,27 @@
 public class NetworkReceiver extends BroadcastReceiver {
 
     private static final String TAG = "NetworkReceiver";
+    private static final int WIFI_AP_STATE_ENABLED = 13;
```

Comment:
Ah that sucks. We should verify that this works accross Android versions. [These](http://stackoverflow.com/a/12531759) [links](http://stackoverflow.com/a/14405339) suggest that the values are different before ICS.